src/main/java/net/authorize/util/XmlUtility.java

```diff
@@ -23,6 +24,9 @@
 public final class XmlUtility {
 	private static Log logger = LogFactory.getLog(XmlUtility.class);
 	private static final String XmlHeader = "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?>";
+    private static JAXBContext request_ctx = null;
+    private static JAXBContext response_ctx = null;
+    private static HashMap<String, JAXBContext> jaxbContext = new HashMap<String, JAXBContext>();
 

```

Comment:
as far as i remember, this class was used only for logging purposes, tht too in debug mode, not in the actual api calls main flow. so, if we wrap the logger calls in ifDebugEnabled, we would not spend time serializing/-de-serializing the request/response unless we are in debug mode.